.gitattributes

```diff
@@ -1 +1 @@
-src/esthri/tests/data/** filter=lfs diff=lfs merge=lfs -text
```

Comment:
I think I might of fudged up the lfs stuff by not using `git mv` to move these files